src/calendar-heatmap.js

```diff
@@ -62,14 +91,40 @@ function calendarHeatmap() {
     return chart;
   };
 
-  function chart() {
+  chart.onMouseOver = function (value) {
+    if (!arguments.length) { return onMouseOver(); }
+    onMouseOver = value;
+    return chart;
+  };
 
+  chart.onMouseOut = function (value) {
+    if (!arguments.length) { return onMouseOut(); }
```

Comment:
Why are you executing `onMouseOut`/`onMouseOver` instead of returning it directly?